flink-yarn-tests/src/test/java/org/apache/flink/yarn/YarnTestBase.java

```diff
@@ -396,11 +397,12 @@ YarnClusterDescriptor createYarnClusterDescriptor(
     YarnClusterDescriptor createYarnClusterDescriptorWithoutLibDir(
             org.apache.flink.configuration.Configuration flinkConfiguration) {
         final YarnClusterDescriptor yarnClusterDescriptor =
-                YarnTestUtils.createClusterDescriptorWithLogging(
-                        tempConfPathForSecureRun.getAbsolutePath(),
-                        flinkConfiguration,
+                new YarnClusterDescriptor(
+                        YarnLogConfigUtil.setLogConfigFileInConfig(
```

Comment:
This is a simple refactoring to use production code. TBH I'm not sure why `YarnTestUtils.createClusterDescriptorWithLogging` exists in the first place.